src/AdminAssets.php

```diff
@@ -14,6 +16,10 @@ class AdminAssets
      */
     public static function enqueue_js(): void
     {
+        if (MediaArchive::is_active()) {
```

Comment:
I need help understanding how the JS is loaded here because if I add this check, the Media Archive does not load for the admin panel, this code `const currentBlock = wp.data.select('core/block-editor').getSelectedBlock();` causes the component to throw an error, if I wrap that code in a `useEffect` to display only in the editor, strangely enough it returns undefined, any thoughts or ideas on why this may be?

Reply:
We can do that in the next iteration when we remove the feature flag.

Reply:
I had a DM with Osong on this comment, we can drop this suggestion, As we are using the same component for both the admin dashboard and the editor.